### PR TITLE
MINOR: Deduplicating ConsumerGroupCommand print formating

### PR DESCRIPTION
**Pull request by nizhikov**
ConsumerGroupCommand contains code duplications for table row format.
This PR reduces code duplication and make it more clear and easy to understand.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala [hunk 1]
```diff
@@ -110,13 +110,16 @@ object ConsumerGroupCommand extends Logging {
   }
 
   def printOffsetsToReset(groupAssignmentsToReset: Map[String, Map[TopicPartition, OffsetAndMetadata]]): Unit = {
-    if (groupAssignmentsToReset.nonEmpty)
-      println("\n%-30s %-30s %-10s %-15s".format("GROUP", "TOPIC", "PARTITION", "NEW-OFFSET"))
+    val format = "%-30s %-30s %-10s %-15s"
+    if (groupAssignmentsToReset.nonEmpty) {
+      println()
```
Comment (hudeqi) [inline core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala]: Why not move this `println()` to `format`？ So is the one below.

Comment (nizhikov) [reply core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala]: Seems like explicit `println` makes it easier to understand what will be printed.
But, of course, we can go with more compact 
`println("\n" + format.format("GROUP", "TOPIC", "PARTITION", "NEW-OFFSET"))`

Comment (nizhikov) [reply core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala]: > Why not move this println() to format？ So is the one below.

Done.

**Review by hudeqi — APPROVED** [review]
LGTM! Thanks for your PR.

**Comment by nizhikov** [comment]
CI failures seems unrelated.

**Comment by nizhikov** [comment]
@hudeqi Are you OK with merging this?

**Comment by hudeqi** [comment]
> @hudeqi Are you OK with merging this?

I'm ok, but need other commiters to merge. @showuon

**Review by showuon — APPROVED** [review]

**Comment by showuon** [comment]
Failed tests are unrelated.

**Pull request merged**